src/components/testimonial.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import Carousel from 'react-bootstrap/Carousel'
+
+const Testimonial = () =>(
+    <div className="container-fluid mt-5 mb-5" id="testimonial">
+        <h2>What Others say about my creations</h2>
+    <Carousel>
+        <Carousel.Item>
+    
+            
+            <p>Thank you soo much. The cake was soo good and it was the best of all! - by Katuuo.</p>
+    
+  </Carousel.Item>
+  <Carousel.Item>
+
+      
+      <p>Priyanga... just want to say thank you very much for the gorgeous cake for my daughter!!! Everyone loved it and the cake was so delicious! Going to pass your name along to my friends if you don't mind! 😃 - by Julie</p>
+    
+  </Carousel.Item>
+  <Carousel.Item>
+
+    
+      
+      <p>Thank you for the beautiful cake Priyanga. It was delicious! 😃 - by Kavitha</p>
```

Comment:
@faithghlee : We could safely remove the emoji's I think; as only the content matters. 😄